apps/financial-aid/web-veita/src/components/TableBody/TableBody.tsx

```diff
@@ -66,7 +66,7 @@ const TableBody = ({ application, index }: PageProps) => {
           })}
         >
           <Text>
-            {months[parseInt(format(new Date(application.created), 'MM'))]}
+            {months[parseInt(format(new Date(application.created), 'MM')) - 1]}
```

Comment:
Does this work instead: 
`months[new Date(application.created).getMonth()]` ?

Reply:
Yess, it works much better! Thanks 🤩 🤩 !!